### PR TITLE
Symfony 5

### PR DESCRIPTION
**Pull request by xvilo**
For projects wanting to migrate to symfony 5, this change is needed. 

If you like to add this while it's not merged, please add this to your composer.json:

```json
"repositories": [
        {
            "type": "vcs",
            "url": "git@github.com:xvilo/HashidsBundle.git"
        }
    ],
```

And update version string like so:

```json
"require": {
        "php": "^7.1.3",
        "roukmoute/hashids-bundle": "dev-symfony-5"
    },
```

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "php": ">=7.1",
     "hashids/hashids": "^3.0",
     "sensio/framework-extra-bundle": "^5.1",
-    "symfony/http-kernel": "^4.0"
```
Comment (roukmoute) [inline composer.json]: Add retro-compatibility:

```json
    "symfony/http-kernel": "^4.0|^5.0"
```

Comment (xvilo) [reply composer.json]: done

**Comment by xvilo** [comment]
@roukmoute when will this be shipped as a next release?

**Comment by roukmoute** [comment]
New version is available: https://github.com/roukmoute/HashidsBundle/releases/tag/v2.3.1

**Comment by xvilo** [comment]
Thanks for the fast merge and version @roukmoute! It's appreciated <3

**Pull request merged**